<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HumanToEpoch</class>
 <widget class="QWidget" name="HumanToEpoch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>From human readable time to epoch time.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConvert">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>27</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>5</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>1770</year>
     <month>9</month>
     <day>14</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>M/d/yyyy hh:mm:ss </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Human-readable time (UTC)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Epoch time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>381</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
